akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/TestProbeSpec.scala

```diff
@@ -142,6 +143,28 @@ class TestProbeSpec extends ScalaTestWithActorTestKit with WordSpecLike {
       }
     }
 
+    "allow receiving one message of type TestProbe[M]" in {
+      val probe = createTestProbe[EventT]()
+      eventsT(10) forall { e ⇒
+        probe.ref ! e
+        probe.receiveOne == e
+      } should ===(true)
+
+      probe.expectNoMessage()
+    }
+
+    "timeout if expected single message is not received" in {
+      intercept[AssertionError](createTestProbe[EventT]().receiveOne)
```

Comment:
here it would be useful to test with a shorter `max` timeout, like `100.millis`, to avoid wasting test running time